plugin/wavesurfer.regions.js

```diff
@@ -424,12 +424,12 @@ WaveSurfer.Region = {
 
                     // Drag
                     if (my.drag && drag) {
```

Comment:
This is a problem I missed. 'my.drag' is checking if the current region being dragged. The names drag() and resize() will break the events bound to that region. Maybe something more like dragRegion() and resizeRegion()?